lib/aasm/instance_base.rb

```diff
@@ -14,15 +14,13 @@ def current_state
 
     def current_state=(state)
       @instance.aasm_write_state_without_persistence(state, @name)
-      # @current_state = state
     end
 
     def enter_initial_state
       state_name = determine_state_name(@instance.class.aasm(@name).initial_state)
       state_object = state_object_for_name(state_name)
 
       state_object.fire_callbacks(:before_enter, @instance)
-      # state_object.fire_callbacks(:enter, @instance)
```

Comment:
@alto Did you planned to remove `enter` callback for state ? 

I can see we are still calling `enter` callback https://github.com/aasm/aasm/blob/master/lib/aasm/aasm.rb#L145 with a TODO: remove for AASM 4.

Should we remove `enter` callback from AASM ?